catkin_ws/src/planner/src/mission.py

```diff
@@ -109,17 +118,5 @@ def master_planner():
     rospy.init_node('mission_planner',log_level=rospy.DEBUG)
     rospy.on_shutdown(endMission)
 
-    try:
-        # mapping = ObjectMapper()
-        state = StateTracker()
-        control = Controller(rospy.Time(0))
-
-        QualiMission()
-
-
-        # ----- UNCOMMENT BELOW TO RUN MISSION(S) -----
-        #testRotationsMission()
-        #laneMarkerGridSearchMission()
-    except KeyboardInterrupt:
-        endMission("Mission end prompted by user. Killing.")
-        exit()
+    control = Controller(rospy.Time(0))
```

Comment:
leave the try/except statement, it is useful to kill missions without having to constantly handle keyboard exceptions at different levels